ghostly.py

```diff
@@ -162,6 +172,29 @@ def switchTo(self, selector):
         self.browser.switch_to_frame(selector)
 
 
+    def assertElement(self, selector):
+        """
+        Assert that an element exists on currently displayed page.
+        """
+        self.wait(1)
+        element = self._get_element(selector)
+        assert element.tag_name == selector
+
+
+    def assertValue(self, *contents):
+        """
+        Assert that an element exists on currently displayed page.
+        """
+        self.wait(1)
+        element = self._get_element(contents[0]['element'])
+        value = contents[0]['value']
+
+        try:
+            element.get_attribute("tagName") is "INPUT"
+        except:
+            print "✘ Element is not an input"
```

Comment:
Probably need to refactor the way that the main loop works though. Ideally this would raise an exception (custom?) that we catch higher up and display the error.